Resume.html

```diff
@@ -0,0 +1,181 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/resume.css">
+    <title>resume</title>
+</head>
+
+<body>
+    <div class="contener">
+        <div class="text_area_nav">
+            <div class="text_nav1">
+                <div class="scouer"></div>
+
+                <h1 class="title_nav">Maya Nelson</h1>
+            </div>
+            <div class="text_nav2">
+                <h2>PROJECT MANAGER</h2>
+            </div>
+
+            <div class="menu_bar">
+                <input id="toggle" type="checkbox"></input>
+                <label for="toggle" class="hamburger">
+                    <div class="top-bun"></div>
+                    <div class="meat"></div>
+                    <div class="bottom-bun"></div>
+                </label>
+
+                <div class="nav">
+                    <div class="nav-wrapper">
+                        <nav>
+                            <a href="./index.html">ABOUT ME</a><br>
+                            <a href="./resume.html">RESUME</a><br>
+                            <a href="./projects.html">PROJECT</a><br>
+                            <a href="./contact.html">CONTACT</a>
+                        </nav>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <main>
+            <div class="titil_resume">
+                <div class="scouer marg a"></div>
+                <h1>Resume</h1>
+            </div>
+            <div class="experience">
+                <h1 class="experience_text">Experience</h1>
+                <button>DOWNLOAD CV</button>
+            </div>
+            <section class="card_1">
+                <div class="all_project">
+                    <div class="title_card">
+                        <h1 class="text_header"> 2019-Persent</h1>
+                        <h4>JOB POSITION
+                        </h4>
+                    </div>
+                </div>
+                <p>I'm a paragraph. Click here to add your own text and edit me. It’s easy. Just click “Edit Text” or
```

Comment:
This text does not look very clear because of the colour contrast, so try to make it bolder maybe or try to make it more clear so that it can be read easily please!